GraphWebhooks/Helpers/GraphHttpClient.cs

```diff
@@ -0,0 +1,72 @@
+﻿
+namespace GraphWebhooks.Helpers
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Net.Http;
+    using System.Net.Http.Headers;
+    using System.Threading.Tasks;
+
+    public class GraphHttpClient
+    {
+        private static readonly int MaxRetryAttempts = 5;
+
+        private string accessToken = null;
+
+        public GraphHttpClient(string accessToken)
+        {
+            this.accessToken = accessToken;
+        }
+
+        // Send HTTP request implemenation with retries on transient failures.
+        public async Task<HttpResponseMessage> SendAsync(HttpRequestMessage request)
```

Comment:
Really like this generic implementation and how this takes an HttpRequestMessage instead of individual params like URL, http method, etc.  Maybe also use GraphHttpClient in NotificationController.cs? When Dan Kershaw points folks to this repo they might see that first and then get confused on where the retry logic is

Reply:
Updated the NotificationController.cs